src/bank/use-cases/find-all/find-all.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { IBankRepository } from '@/bank/interfaces';
+import { FindAllBanksUseCase } from '@/bank/use-cases';
+import { Bank } from '@/bank/infra/entities';
+
+describe('Find Bank by Id', () => {
+  let findAll: FindAllBanksUseCase;
+  let bankRepository: jest.Mocked<IBankRepository>;
+
+  beforeAll(() => {
+    bankRepository = {
+      findAll: jest.fn(),
+    } as unknown as jest.Mocked<IBankRepository>;
+
+    findAll = new FindAllBanksUseCase(bankRepository);
+  });
+
```

Comment:
Por fim um teste um pouco inútil

it('should be defined', () => {
    expect(bankRepository).toBeDefined();
    expect(findAll).toBeDefined();
  });

mas valido